csxtools/lmfit_models.py

```diff
@@ -0,0 +1,200 @@
+import numpy as np

```

Comment:
@tacaswell Would you like to add this to scikit-beam? They have been pretty handy so far for fitting 2D data. I can add numpy style docstrings if you do decide to include it.